ow_plexil/src/plans/FaultHandlingPattern4.plp

```diff
@@ -0,0 +1,81 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+// If no faults are occurring, this plan will pan 180 degrees, 
+// and then unstow and stow the arm.
+// If any fault is injected, the plan will pause and wait for resolution.
+// If a fault is injected during an operation, that operation will only
+// halt immediately if the fault injected is associated with it directly,
+// i.e. an antenna fault during panning or an arm fault during 
+// unstowing or stowing.
+// Otherwise, the pausing will visually become apparent when the next
+// operation does not begin due to it.
+
+#include "plan-interface.h"
+
+LibraryAction InitializeAntennaAndArm ();
+
+FaultHandlingPattern4:
+{
+
+  Boolean PanDone = false;
+  Boolean UnstowDone = false;
+  Boolean StowDone = false;
+
+  Real AngleToGoal;
```

Comment:
This should should ideally be defined in PanAntenna, as that is the only place it's used.  There, I'll suggest an alternate formula altogether.